transformer.py

```diff
@@ -75,6 +75,7 @@ def process(self, name):
         :return dict: Merged data.
         """
         trfmdata = []
+        self.files.sort()
```

Comment:
It's better to do this sorting here since we don't want to let users override it: https://github.com/mozilla/mach-perftest-notebook-dev/blob/starter/perftestnotebook.py#L102

```
if self.sort_files:
    if type(files) == list:
        files.sort()
    else:
        for _, file_list in files:
            file_list.sort()
```